OpenRA.Game/Game.cs

```diff
@@ -392,6 +393,17 @@ public static void InitializeMod(string mod, Arguments args)
 
 			JoinLocal();
 
+			try
+			{
+				discoverNat.Wait();
+			}
+			catch (Exception e)
+			{
+				Console.WriteLine("NAT discovery failed: {0}", e.Message);
+				Log.Write("nat", e.StackTrace);
```

Comment:
I guess the exception message would be better.

In my case that only contained `NAT discovery failed: Object reference not set to an instance of an object`, though.

Erm, added some more checking there. It seems `discoverNat` is null in the `try` block. No wonder it throws :p